Week1/homework/6-array-of-animals.js

```diff
@@ -0,0 +1,8 @@
+"use strict";
+let animalsArr;
+console.log("I think the value will be undefined");
+console.log(animalsArr);
+let favoriteAnimals = ["Cats", "Dogs", "Birds",];
+console.log(favoriteAnimals);
+favoriteAnimals.push("Piglet");
+console.log(favoriteAnimals);
```

Comment:
Ah. You were supposed to write as comments what you think the new value of `favoriteAnimals` will be. 🙂